### DISCUSSION TITLE
Test failed

### DISCUSSION THREAD

**Discussion by Wolke24** [idaholab/moose#23033] [Q&A Getting Started] [answered]
At the last step of the installation, I used the following commands:
cd ~/projects/moose/test
make -j 4
./run_tests -j 4
Here is the issue report:

<img width="848" alt="Screen Shot 2022-12-27 at 10 32 01 PM" src="https://user-images.githubusercontent.com/103281861/209757696-6fab9b15-8e77-420c-b4b2-64c5765204c3.png">

Does anyone know the cause of this problem？

**Comment by GiudGiud** [comment, marked as answer]
Hello

All these tests use MPI.
Are you running this on a mac?
What do you get when you run 
`mpirun -n 4 <moose_executable> -i test/tests/kernels/simple_diffusion/simple_diffusion.i` ?

I think you likely will find the solution to this issue here:
https://mooseframework.inl.gov/help/troubleshooting.html#811225ec-4f15-4717-99a2-0035ee939de4

Guillaume

**Reply by Wolke24** [reply]
Hi Guillaume,
yes, I am using a Mac. But when I run the command you give, it says: no such file or directory: moose_executable

**Reply by GiudGiud** [reply]
oh I meant moose-executable as a placeholder. 
You should use the test executable, should be something like
mpirun -n 4 moose/test/moose_test-opt -i test/tests/kernels/simple_diffusion/simple_diffusion.i ?

**Reply by Wolke24** [reply]
the -j 8 test also fails.
<img width="1084" alt="Screen Shot 2022-12-28 at 1 50 26 PM" src="https://user-images.githubusercontent.com/103281861/209865021-9541cde1-1a69-4b28-a00c-66fa42e93f92.png">
so I follow the guides for "build issues", but when I run the hello world example, MPI issues also occur.
<img width="581" alt="Screen Shot 2022-12-28 at 1 53 47 PM" src="https://user-images.githubusercontent.com/103281861/209865359-e5e25202-cd19-4922-bc70-cdca16e4bda2.png">

**Reply by GiudGiud** [reply]
ok so in the troubleshooting page we have this exact error
please look for gethostbyname on that page

**Reply by Wolke24** [reply]
Thank you! after I edited the host file the tests passed!
<img width="766" alt="Screen Shot 2022-12-28 at 6 58 18 PM" src="https://user-images.githubusercontent.com/103281861/209890208-4d560f9f-9c6d-4a1c-8f87-c6f389f401ee.png">

but there are some warnings do you think it's necessary to solve them?

<img width="1158" alt="Screen Shot 2022-12-28 at 6 36 23 PM" src="https://user-images.githubusercontent.com/103281861/209890414-6eaaf1b7-93ba-4e75-9a18-92a3c8bc6d12.png">

**Reply by GiudGiud** [reply]
no need to worry about those
